test/__snapshots__/index.test.js.snap

```diff
@@ -1194,30 +1193,30 @@ class-decorators-ts
 
 
 Build \\"class-decorators-ts\\" to dist:
-348 B: class-decorators-ts.js.gz
-288 B: class-decorators-ts.js.br
-348 B: class-decorators-ts.esm.mjs.gz
-288 B: class-decorators-ts.esm.mjs.br
-415 B: class-decorators-ts.umd.js.gz
-353 B: class-decorators-ts.umd.js.br"
+370 B: class-decorators-ts.js.gz
+311 B: class-decorators-ts.js.br
+367 B: class-decorators-ts.esm.mjs.gz
+302 B: class-decorators-ts.esm.mjs.br
+435 B: class-decorators-ts.umd.js.gz
+380 B: class-decorators-ts.umd.js.br"
 `;
 
 exports[`fixtures build class-decorators-ts with microbundle 2`] = `7`;
 
 exports[`fixtures build class-decorators-ts with microbundle 3`] = `
-"var e=/*#__PURE__*/function(){function e(e){this.greeting=e}return e.prototype.greet=function(){return\\"Hello, \\"+this.greeting},e}(),t=new(e=function(e,t,r,n){var o,c=arguments.length,l=c<3?t:null===n?n=Object.getOwnPropertyDescriptor(t,r):n;if(\\"object\\"==typeof Reflect&&\\"function\\"==typeof Reflect.decorate)l=Reflect.decorate(e,t,r,n);else for(var f=e.length-1;f>=0;f--)(o=e[f])&&(l=(c<3?o(l):c>3?o(t,r,l):o(t,r))||l);return c>3&&l&&Object.defineProperty(t,r,l),l}([function(e){Object.seal(e),Object.seal(e.prototype)}],e))(\\"Hello World\\");export default t;
+"\\"function\\"==typeof SuppressedError&&SuppressedError;var e=/*#__PURE__*/function(){function e(e){this.greeting=e}return e.prototype.greet=function(){return\\"Hello, \\"+this.greeting},e}(),t=new(e=function(e,t,r,o){var n,c=arguments.length,f=c<3?t:null===o?o=Object.getOwnPropertyDescriptor(t,r):o;if(\\"object\\"==typeof Reflect&&\\"function\\"==typeof Reflect.decorate)f=Reflect.decorate(e,t,r,o);else for(var l=e.length-1;l>=0;l--)(n=e[l])&&(f=(c<3?n(f):c>3?n(t,r,f):n(t,r))||f);return c>3&&f&&Object.defineProperty(t,r,f),f}([function(e){Object.seal(e),Object.seal(e.prototype)}],e))(\\"Hello World\\");export default t;
```

Comment:
this is weird. I wonder why it's being injected? it's dead code.

Reply:
Comes from [`tslib`](https://github.com/microsoft/tslib/blob/d72d6f70b36286bc3f94a3dda1e64dcb568b1370/tslib.es6.mjs#L328) which seems to treat it like a global. Not quite sure why terser keeps it around.

Reply:
It looks like we can strip this out via `compress.global_defs`:

```
compress: {
  global_defs: {
    SuppressedError: 'undefined'
  },
}
```

<img width="978" alt="Screenshot 2025-01-07 at 8 59 13 PM" src="https://github.com/user-attachments/assets/96b9ac70-20d6-48b0-aaf6-0a057ff4fcef" />